v4.3.1: add release notes (#33)
bisdn/bisdn-linux: release-notes.txt

BISDN Linux
4.3.1 Release Notes
Date: 2021-11-30

! Changed behaviour

> - Certain utilities like client_drivshell will now require root privileges

Reviewer: we might want to highlight here the severeness of what was fixed

BISDN Linux
4.3.1 Release Notes
Date: 2021-11-30

! Changed behaviour
- Certain utilities like client_drivshell will now require root privileges

! Bug fixes
- Validate stp arguments in ofdpa-grpc instead of passing them on as is

! Security fixes

> - Fixed an issue where unauthenticated users could inject shell commands
>   executed with root privileges over the network
> - Fixed an issue where local users could inject shell commands executed
>    with root privileges

Reviewer: ```suggestion
- Fixed issues where users could inject shell commands either locally or unauthenticated over the network 
   being executed with root privileges
```

Author: Not sure if it makes sense merging these two distinct issues. If we do, I would at least move the executed with root privileges to the shell commands, and maybe s/users/actors/.
e.g.
```suggestion
- Fixed issues allowing actors to inject shell commands executed with root privileges locally and unauthenticated over the network.
```

Reviewer: sounds good
